### PR TITLE
[src/timer_irq] add the fixed comment

### PR DESCRIPTION
**Pull request by supperthomas**
## 拉取/合并请求描述：(PR description)

[add the fixed comment]

以下的内容不应该在提交PR时的message修改，修改下述message，PR会被直接关闭。请在提交PR后，浏览器查看PR并对以下检查项逐项check，没问题后逐条在页面上打钩。
The following content must not be changed in the submitted PR message. Otherwise, the PR will be closed immediately. After submitted PR, please use a web browser to visit PR, and check items one by one, and ticked them if no problem.

### 当前拉取/合并请求的状态 Intent for your PR

必须选择一项 Choose one (Mandatory):

- [ ] 本拉取/合并请求是一个草稿版本 This PR is for a code-review and is intended to get feedback
- [x] 本拉取/合并请求是一个成熟版本 This PR is mature, and ready to be integrated into the repo

### 代码质量 Code Quality：

我在这个拉取/合并请求中已经考虑了 As part of this pull request, I've considered the following:

- [x] 已经仔细查看过代码改动的对比 Already check the difference between PR and old code
- [x] 代码风格正确，包括缩进空格，命名及其他风格 Style guide is adhered to, including spacing, naming and other styles
- [x] 没有垃圾代码，代码尽量精简，不包含`#if 0`代码，不包含已经被注释了的代码 All redundant code is removed and cleaned up
- [x] 所有变更均有原因及合理的，并且不会影响到其他软件组件代码或BSP All modifications are justified and not affect other components or BSP
- [x] 对难懂代码均提供对应的注释 I've commented appropriately where code is tricky
- [x] 本拉取/合并请求代码是高质量的 Code in this PR is of high quality
- [x] 本拉取/合并符合[RT-Thread代码规范](../documentation/coding_style_cn.md) This PR complies with [RT-Thread code specification](../documentation/coding_style_en.txt)

### CODE REVIEW

**Review by mysterywolf — APPROVED** [review]

File: src/timer.c [hunk 1]
```diff
@@ -218,12 +223,18 @@ void rt_timer_dump(rt_list_t timer_heads[])
...
  * @brief This function will initialize a timer
  *        normally this function is used to initialize a static timer object.
- * @param timer the static timer object
- * @param name the name of timer
- * @param timeout the timeout function
- * @param parameter the parameter of timeout function
- * @param time the tick of timer
- * @param flag the flag of timer
+ *
+ * @param timer The static timer object
```
Comment (mysterywolf) [inline src/timer.c:227]: timer is the static timer object. 介绍param的时候开头小写 通过be动词连接
相似地：
name is the name of timer

**Review by mysterywolf — CHANGES_REQUESTED** [review]

File: src/timer.c [hunk 2]
```diff
@@ -72,9 +72,9 @@ void rt_timer_enter_sethook(void (*hook)(struct rt_timer *timer))
 
 /**
  * @brief This function will set a hook function, which will be
- * invoked when exit * timer timeout callback function.
+ *        invoked when exit timer timeout callback function.
  *
- * @param hook the function point of timer
+ * @param hook The function point of timer
```
Comment (mysterywolf) [inline src/timer.c:77]: param 的句子不需要大写，以param为主语 直接通过be动词连接说明即可 详见ipc 除了param的描述，其他的都需要大写开头首字母。

File: src/timer.c [hunk 3]
```diff
@@ -126,11 +130,11 @@ static void _timer_init(rt_timer_t timer,
...
 
 /**
- * @brief  find the next emtpy timer
+ * @brief  Find the next emtpy timer ticks
  *
- * @param timer_list the timer of the next timeout
+ * @param timer_list The timer of the next timeout
  *
- * @return rt_tick_t the point of timer
+ * @return rt_tick_t The next timer's ticks
```
Comment (mysterywolf) [inline src/timer.c:137]: return的时候不需要强调数据类型，直接描述返回值的功能即可
例如：
Return the next timer's ticks.

**Review by mysterywolf — CHANGES_REQUESTED** [review]

**Review by supperthomas — COMMENTED** [review]
resloved

**Review by supperthomas — COMMENTED** [review]
@mysterywolf

**Review by mysterywolf — APPROVED** [review]

**Pull request merged**